components/Pagination.js

```diff
@@ -0,0 +1,148 @@
+import { useEffect, useState } from "react";
+import { FaChevronRight } from "react-icons/fa";
+import { FaChevronLeft } from "react-icons/fa";
+import { BsChevronDoubleRight } from "react-icons/bs";
+import { BsChevronDoubleLeft } from "react-icons/bs";
```

Comment:
I just need to check these imports are tree shaken, as the other icon sets by `react-icons` were not and included 10k+ files

Reply:
I switched icon imports react-icons to hero-icons/react,
```js
import ChevronDoubleLeftIcon from "@heroicons/react/24/outline/ChevronDoubleLeftIcon";
import ChevronDoubleRightIcon from "@heroicons/react/24/outline/ChevronDoubleRightIcon";
import ChevronRightIcon from "@heroicons/react/24/outline/ChevronRightIcon";
import ChevronLeftIcon from "@heroicons/react/24/outline/ChevronLeftIcon";
```
it should be good to go now.
![Screenshot build](https://github.com/EddieHubCommunity/LinkFree/assets/73095526/f35cce92-9ed7-4547-9c78-e807392fc4e1)